app/javascript/pages/listings/listing-detail.tsx

```diff
@@ -1,37 +1,36 @@
-import React, { useContext, useState, useEffect } from "react"
+import React, { useContext, useEffect, useState } from "react"
 import dayjs from "dayjs"
 import {
-  SiteAlert,
-  ListingDetails,
-  ListingDetailItem,
-  ImageCard,
-  Message,
+  AdditionalFees,
   ApplicationStatus,
-  Waitlist,
+  ApplicationStatusType,
+  ContentAccordion,
+  Description,
   EventSection,
-  ListSection,
-  PreferencesList,
-  InfoCard,
   ExpandableText,
-  Description,
-  AdditionalFees,
+  ImageCard,
+  InfoCard,
+  ListingDetailItem,
+  ListingDetails,
+  ListSection,
+  LoadingOverlay,
   MarkdownSection,
+  Message,
   NavigationContext,
-  LoadingOverlay,
-  ContentAccordion,
+  PreferencesList,
+  SiteAlert,
+  t,
+  Waitlist,
 } from "@bloom-housing/ui-components"
 
 import Layout from "../../layouts/Layout"
 import withAppSetup from "../../layouts/withAppSetup"
 import { getListing } from "../../api/listingApiService"
 import { getReservedCommunityType } from "../../util/languageUtil"
-import {
-  RailsListing,
-  getListingImageCardStatuses,
-  getListingAddressString,
-} from "../../modules/listings/SharedHelpers"
+import { getListingAddressString, RailsListing } from "../../modules/listings/SharedHelpers"
 import { ListingEvent } from "../../api/types/rails/listings/BaseRailsListing"
 
+const alertIconColor = "#E31C3D"
```

Comment:
I'm new to tailwind. Would appreciate any help on how to access the configured palette in react.

Reply:
[here's](https://github.com/bloom-housing/bloom/blob/dev/ui-components/tailwind.config.js#L35-L75) the color tokens from the bloom theme. I don't see `#E31C3D` in there, though the difference between that and `red.700` (`#E41D3D`) is negligible. if it's fine to roll with that, I think you'd wanna do something like
```suggestion
import bloomTheme from "../../../../tailwind.config"
...
iconColor={!isApplicationOpen && bloomTheme.theme.colors.red.700}
...
```